homeassistant/components/private_ble_device/sensor.py

```diff
@@ -83,7 +83,8 @@ class PrivateDeviceSensorEntityDescription(
         native_unit_of_measurement=UnitOfTime.SECONDS,
         entity_registry_enabled_default=False,
         entity_category=EntityCategory.DIAGNOSTIC,
-        value_fn=lambda hass, service_info: bluetooth.async_get_learned_advertising_interval(
+        value_fn=lambda hass,
```

Comment:
This casuses a multiline lambda, which should add some parenthesis, instead of accepting this formatting